java/client/src/main/java/glide/api/models/commands/InfoOptions.java

```diff
@@ -0,0 +1,63 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Singular;
+
+/**
+ * Optional arguments to ServerCommands#info(InfoOptions)
+ *
+ * @see <a href="https://redis.io/commands/info/">redis.io</a>
+ */
+@Builder
+public final class InfoOptions {
```

Comment:
can this be removed from this PR? you can add it after the info command PR is merged.

Reply:
I was actually intending to merge this _after_ the info() command PR.  There's no rush on this one.